CONTRIBUTING.md

```diff
@@ -24,7 +26,8 @@ side of including lots of information, such as:
   chance to talk it over with the owners and validate your approach.
 * Nox maintains 100% test coverage. All pull requests must maintain this.
 * Follow [pep8](https://pep8.org).
-* Update documentation, if relevant.
+* Update documentation and tests if relevant.
+* Ensure your changes pass nox's tests and lint suites before pushing.
```

Comment:
Since we are referring to the Nox project, I suggest a spelling change (if not, just ignore this):

```suggestion
* Ensure your changes pass Nox's tests and lint suites before pushing.
```